staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -88,6 +88,9 @@ type Event struct {
 
 	// RequestURI is the request URI as sent by the client to a server.
 	RequestURI string
+	// UserAgent is an optional field that specifies the caller of this request.
```

Comment:
Please add a note that this is trivially spoofable. e.g.
```
// Note that the UserAgent is provided by the client, and must not be trusted.
```

Reply:
ok